src/lib/y2storage/autoinst_profile/partition_section.rb

```diff
@@ -288,6 +291,23 @@ def init_mount_options(fs)
         @fstab_options = fs.fstab_options unless fs.fstab_options.empty?
       end
 
+      # @param fs [Filesystem::BlkFilesystem]
```

Comment:
It is obvious, but maybe to add a comment saying that it populates `@subvolumes` ¿?